lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -615,17 +615,17 @@ namespace Js
                         recycler->ReportExternalMemoryFailure(length);
                     }
                 }
-                else
-                {
-                    JavascriptError::ThrowOutOfMemoryError(GetScriptContext());
-                }
             }
 
             if (buffer != nullptr)
             {
                 bufferLength = length;
                 ZeroMemory(buffer, bufferLength);
             }
+            else
```

Comment:
Yes

---
In reply to: [128093238](https://github.com/Microsoft/ChakraCore/pull/3384#discussion_r128093238) [](ancestors = 128093238)